### PR TITLE
Resolve #1530, #1532, #1586, #1585, #1599, #1596, #1598, #1533, #1595, #1600, #1601, #1605, #1606

### LINKED ISSUE #1530: [Test] Test receiver email for cancelTransfer tests

### LINKED ISSUE #1532: [Test] Test sender email for cancelTransfer tests

### PR DESCRIPTION
**Pull request by InVincible2016**
- Resolve #1530 [Test] Test receiver email for cancelTransfer tests
- Resolve #1532 [Test] Test sender email for cancelTransfer tests

### CODE REVIEW

**Comment by InVincible2016** [comment]
Screen Shot 2020-05-07 at 5 41 10 PM

File: src/actions/userActions.js [hunk 1]
```diff
@@ -248,10 +248,14 @@ function postLoginPreparation (loginData: any, progress?: Function) {
     dispatch(onGoogleLoginReturn(loginData))
     return dispatch({
       type: 'POST_LOGIN_PREPARATION',
-      payload: new Promise(async (resolve, reject) => {
-        // register/get user
+      payload: async (resolve, reject) => {
```
Comment (timothywangdev) [inline src/actions/userActions.js:251]: remove resolve & reject since they are redundant

Comment (timothywangdev) [reply src/actions/userActions.js:251]: 1

Comment (InVincible2016) [reply src/actions/userActions.js:251]: True, typo, fixed

**Pull request merged**